File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
  const reg = /^(aya|ye|woo|ma)+$/;

  const res = babbling.reduce((count, babu) => (reg.test(babu) ? count + 1 : count), 0);
  
  const possible = ["aya", "ye", "woo", "ma" ]
  
  let arr = []
  
   for(let b of babbling) {
        for(const p of possible) {
             b = b.replace(p, "_")
   
        }
    arr.push(b)
   }
    
    const test = arr.map(str => str.split('').every(char => char === '_'))
    const count = test.filter(v => v).length
    console.log(count)
    
  return count;
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0 
    
    print(1)
    
    babble = ["aya", "ye", "woo", "ma"]
    for bab_list in babbling:
        for text in babble:
            if text * 2 not in bab_list:
                bab_list = bab_list.replace(text, " ")
        if bab_list.strip() == "":
            answer += 1
    return answer